client/my-sites/people/people-profile/index.jsx

```diff
@@ -77,6 +79,23 @@ class PeopleProfile extends React.PureComponent {
 		return 'role-' + role;
 	};
 
+	handleLinkToReaderSiteStream = event => {
+		const modifierPressed =
+			event.button > 0 || event.metaKey || event.controlKey || event.shiftKey || event.altKey;
+
+		recordTrack( 'calypso_sites_people_followers_link_click', {
+			modifier_pressed: modifierPressed,
+		} );
+
+		if ( modifierPressed ) {
+			return;
+		}
+
+		const blogId = get( this.props.user, 'follow_data.params.blog_id', false );
```

Comment:
should probably bail if we get `false` here

Reply:
i already have the bail in calling code, but makes sense to have this fully independent

Reply:
bail added! ready for final 👀